package.json

```diff
@@ -11,9 +11,10 @@
     "test-es2015-headless": "ng test angular-cld --tsConfig=projects/angular-cld/tsconfig-es2015.spec.json --browsers ChromeHeadless",
     "install-sample-from-source": "node ./scripts/install-sample-from-source.js samples/photo_album",
     "install-sample-from-source:jquery": "node ./scripts/install-sample-from-source.js samples/photo_album_with_jquery",
-    "build": "ng build angular-cld && npm run copy-lib-assets",
+    "build": "ng build angular-cld && npm run copy-lib-assets && npm run prepare-version",
```

Comment:
Does that mean we need to run 'build' before we run anything else? otherwise things will not work

Reply:
This is true always. We always need to build the project for it to work after every change we make

Reply:
Why do we need to prepare the version before the build?
Isn't it the other way around?
First, prepare the version then build?